library/nagvis-includes/CoreAuthModIcingaweb2.php

```diff
@@ -126,4 +126,7 @@ public function getUserId()
     {
         return $this->getUser();
     }
+    public function usesBcrypt($username) {
+        return false;
+    }
```

Comment:
Can you please add a new line before this new method and put the opening `{` on the next line as well, just like the other methods? Thank you!